connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/PluginUtils.java

```diff
@@ -18,24 +18,31 @@
 
 import java.io.IOException;
 import java.lang.reflect.Modifier;
-import java.net.URL;
 import java.nio.file.DirectoryStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedList;
 import java.util.List;
 import java.util.Locale;
+import java.util.Set;
 
 public class PluginUtils {
     private static final String BLACKLIST = "^(?:"
             + "java"
             + "|javax"
+            + "|sun"
             + "|org\\.omg"
             + "|org\\.w3c\\.dom"
             + "|org\\.apache\\.kafka\\.common"
             + "|org\\.apache\\.kafka\\.connect"
             + "|org\\.apache\\.log4j"
+            + "|org\\.slf4j"
```

Comment:
Should we be removing the log4j one here since the plugins should really only be relying on slf4j being available?